### PR TITLE
React redux capstone

### PR DESCRIPTION
**Pull request by Mujeeb4582**
# Task Completed

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API and stored in the Redux store.
- Filter the data using a Filter stateless component.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed and accessible online, check in the readme file.
- Also Add the video link in the readme file.

### CODE REVIEW

File: src/__test__/__snapshots__/about.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+  >
+    <h4>
+      United States
+      , 
+      US
+    </h4>
+    <h6>
+      Date: 
+      1/19/1970
+
```
Comment (sja-thedude) [inline src/__test__/__snapshots__/about.test.js.snap]: - [ ] Kindly fix the error which occurred when running npm test.

Screenshot 2023-01-06 at 4 50 55 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Mujeeb4582, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct Gitflow flow
✔️ Good Video Presentation

## Required 
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/__snapshots__/about.test.js.snap (hunk 1 above)
Comment (Mujeeb4582) [reply src/__test__/__snapshots__/about.test.js.snap]: @sja-thedude I am unsure why the some test is not working on your end. Could you please check it again and let me know if it is functioning properly? If it is, I will request another review.
image

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Mujeeb4582, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- All tests are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**